Makefile.pre.in

```diff
@@ -496,6 +496,45 @@ LIBRARY_OBJS=	\
 DTRACE_DEPS = \
 	Python/ceval.o Python/import.o Python/sysmodule.o Modules/gcmodule.o
 
+##########################################################################
+# decimal's libmpdec
+
+LIBMPDEC_OBJS= \
+		Modules/_decimal/libmpdec/basearith.o \
+		Modules/_decimal/libmpdec/constants.o \
+		Modules/_decimal/libmpdec/context.o \
+		Modules/_decimal/libmpdec/convolute.o \
+		Modules/_decimal/libmpdec/crt.o \
+		Modules/_decimal/libmpdec/difradix2.o \
+		Modules/_decimal/libmpdec/fnt.o \
+		Modules/_decimal/libmpdec/fourstep.o \
+		Modules/_decimal/libmpdec/io.o \
+		Modules/_decimal/libmpdec/mpalloc.o \
+		Modules/_decimal/libmpdec/mpdecimal.o \
```

Comment:
I see that `mpsignal.o` is missing here; presumably that's because we don't actually make any use of it? Is the omission worth a comment? (`bench` and `bench_full` are also missing, but the reason for those is more obvious)

Reply:
I didn't include it because ``setup.py``'s ``detect_decimal`` does not include ``mpsignal.o`` either. It doesn't hurt to include the object. The linker should drop any unused symbol from the final library anyway.

``bench`` and ``bench_full`` are programs with ``main`` function. We cannot include them.

Reply:
I added a build rule and comment for ``mpsignal.o``.